src/test/java/com/elasticbox/jenkins/tests/TestUtils.java

```diff
@@ -60,6 +61,11 @@
     static final String DEFAULT_TEST_WORKSPACE = "test_admin";
     static final String TEST_WORKSPACE = System.getProperty("elasticbox.jenkins.test.workspace", DEFAULT_TEST_WORKSPACE);
 
+    static final String TEST_JENKINS_PROXY_NAME = "http://21.22.23.24";
```

Comment:
Since these ones are only required and used by the ProxyTest class, we may move these there as internal constants. What do you think?